resources/lrsql/config/test/maria/database.edn

```diff
@@ -0,0 +1,4 @@
+#merge
+[#include "test/default/database.edn"
```

Comment:
so does that file exist or

Reply:
If you want to preserve the pattern, make an empty `test/default/database.edn` file, as is done for `test/default/tuning.edn`

Reply:
looking at this, I think Kelvin ran in to the same thing, noted that Aero in fact does not care if you try to merge a nonexistent file, then did the same thing I am suggesting here lol